### PR TITLE
Urls ukchina serbian zhongwen test

### PR DESCRIPTION
**Pull request by LilyL0u**
Resolves #NUMBER

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

Running on test now

### CODE REVIEW

File: cypress/support/config/settings.js [hunk 1]
```diff
@@ -3426,11 +3426,11 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/serbian/cyr/23279016'],
             enabled: false,
           },
           local: {
-            paths: [], // Once support for variants is available: '/serbian/cyr/srbija-49427344'
+            paths: ['/serbian/cyr/23279016'], // Once support for variants is available: '/serbian/cyr/srbija-49427344'
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Can we update the comment stating that it's a CPS MAP with video / audio?

File: cypress/support/config/settings.js [hunk 2]
```diff
@@ -4797,11 +4800,14 @@ module.exports = () => ({
...
-            paths: [],
+            paths: [
+              '/ukchina/trad/23279012', // CPS
+              '/ukchina/trad/multimedia/2016/11/161104_tc2_testmap1', // TC2
+            ],
             enabled: false,
           },
           local: {
-            paths: [], // Enable once variants supported '/ukchina/trad/49375846'
+            paths: ['/ukchina/trad/23279012'], // Enable once variants supported '/ukchina/trad/49375846'
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Replace comment with `CPS MAP with video|audio`

Comment (RichardPK) [inline cypress/support/config/settings.js]: Should this comment be removed?

File: cypress/support/config/settings.js (hunk 1 above)
Comment (RichardPK) [inline cypress/support/config/settings.js]: Should this comment be removed?

File: cypress/support/config/settings.js [hunk 3]
```diff
@@ -3537,11 +3537,11 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/serbian/lat/23279016'],
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Can we add a quick comment stating that it's a CPS MAP with video / audio?

File: cypress/support/config/settings.js [hunk 4]
```diff
@@ -3537,11 +3537,11 @@ module.exports = () => ({
...
             enabled: false,
           },
           local: {
-            paths: [],
+            paths: ['/serbian/lat/23279016'],
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Can we add a quick comment stating that it's a CPS MAP with video / audio?

File: cypress/support/config/settings.js [hunk 5]
```diff
@@ -5653,11 +5659,11 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/zhongwen/trad/uk-23283128'],
             enabled: false,
           },
           local: {
-            paths: [], // Once variant support is available '/zhongwen/trad/chinese-news-49631219'
+            paths: ['/zhongwen/trad/uk-23283128'], // Once variant support is available '/zhongwen/trad/chinese-news-49631219'
```
Comment (RichardPK) [inline cypress/support/config/settings.js]: Should this comment be removed?

File: cypress/support/config/settings.js [hunk 6]
```diff
@@ -3426,11 +3426,11 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/serbian/cyr/23279016'],
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Can we add a quick comment stating that it's a CPS MAP with video / audio?

File: cypress/support/config/settings.js [hunk 7]
```diff
@@ -4686,11 +4686,14 @@ module.exports = () => ({
...
-            paths: [],
+            paths: [
+              '/ukchina/simp/23279012', // CPS
+              '/ukchina/simp/multimedia/2016/11/161104_tc2_testmap1', // TC2
+            ],
             enabled: false,
           },
           local: {
-            paths: [],
+            paths: ['/ukchina/simp/23279012'],
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Can we add a quick comment stating that it's a CPS MAP with video / audio?

File: cypress/support/config/settings.js [hunk 8]
```diff
@@ -4686,11 +4686,14 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: [
+              '/ukchina/simp/23279012', // CPS
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Can we add whether is is video/audio in the comment?

**Review by RichardPK — COMMENTED** [review]
Not sure if the commented code needs deleting now?

**Review by ryanmccombe — APPROVED** [review]

File: cypress/support/config/settings.js [hunk 9]
```diff
@@ -4797,11 +4800,14 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: [
+              '/ukchina/trad/23279012', // CPS
+              '/ukchina/trad/multimedia/2016/11/161104_tc2_testmap1', // TC2
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Can we update the comment stating that it's a CPS MAP with video / audio?

**Review by karinathomasbbc — CHANGES_REQUESTED** [review]

File: cypress/support/config/settings.js [hunk 10]
```diff
@@ -5653,11 +5659,11 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/zhongwen/trad/uk-23283128'],
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Can we add a quick comment stating that it's a CPS MAP with video / audio?

File: cypress/support/config/settings.js (hunk 5 above)
Comment (karinathomasbbc) [reply cypress/support/config/settings.js]: Or update the comment stating that it is CPS with video / audio

File: cypress/support/config/settings.js (hunk 10 above)
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: Do we also need TC2 MAPs for zhongwen? `zhongwen/trad/multimedia/2016/11/161107_tc2_testmap1`

File: cypress/support/config/settings.js [hunk 11]
```diff
@@ -5542,11 +5548,11 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/zhongwen/simp/uk-23283128'],
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js]: TC2: `zhongwen/simp/multimedia/2016/11/161107_tc2_testmap1`

**Review by karinathomasbbc — APPROVED** [review]

File: cypress/support/config/settings.js (hunk 2 above)
Comment (LilyL0u) [reply cypress/support/config/settings.js]: removed

File: cypress/support/config/settings.js (hunk 1 above)
Comment (LilyL0u) [reply cypress/support/config/settings.js]: removed

File: cypress/support/config/settings.js (hunk 5 above)
Comment (LilyL0u) [reply cypress/support/config/settings.js]: removed

File: cypress/support/config/settings.js (hunk 10 above)
Comment (LilyL0u) [reply cypress/support/config/settings.js]: added

Comment (LilyL0u) [reply cypress/support/config/settings.js]: added

File: cypress/support/config/settings.js (hunk 11 above)
Comment (LilyL0u) [reply cypress/support/config/settings.js]: added

**Pull request merged**